Manager/src/main/java/io/raspberrywallet/manager/common/http/ApacheHttpClient.java

```diff
@@ -0,0 +1,58 @@
+package io.raspberrywallet.manager.common.http;
+
+import org.apache.commons.lang.NotImplementedException;
+import org.apache.http.Header;
+import org.apache.http.HttpResponse;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.fluent.Form;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.message.BasicHeader;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ApacheHttpClient extends HttpClient {
+    
+    private String host;
+    private Header[] defaultHeaders;
+    
+    public ApacheHttpClient(String host, Form defaultHeaders) {
+        this.host = host;
+        this.defaultHeaders = toHeadersArray(defaultHeaders);
+    }
+    
+    @Override
+    public HttpResponse sendPOSTRequest(Form body, String endpoint) throws IOException {
+        org.apache.http.client.HttpClient client = HttpClients.createDefault();
```

Comment:
It would be nice if you could reuse OkHttpClient with Jackson, it's already in our dependencies, and seems to be much smaller and have nicer API